### PR TITLE
Resolve #1720

### LINKED ISSUE #1720: Add CLI tests for two new parameters in factory.make_environment
```
--location-ids LOCATION_IDS           REPLACE locations with given ids
                                      Comma separated list of values.
--organization-ids ORGANIZATION_IDS   Comma separated list
                                      of values.
```

### PR DESCRIPTION
**Pull request by oshtaier**
Implemented CLI test cases covering two new
parameters in factory.make_environments method

### CODE REVIEW

File: tests/foreman/cli/test_environment.py [hunk 1]
```diff
@@ -83,6 +83,52 @@ def test_list(self):
...
+
+        @Feature: Environment - Create
+
+        @Assert: Environment is created and has new Location assigned
+
+        """
+        name = gen_string("alpha", 10)
+        try:
+            new_loc = make_location()
+            make_environment({
```
Comment (elyezer) [inline tests/foreman/cli/test_environment.py]: Store the environment:

```python
environment = make_environment({
    ...
})
```

File: tests/foreman/cli/test_environment.py [hunk 2]
```diff
@@ -83,6 +83,52 @@ def test_list(self):
...
+                'name': name,
+                'location-ids': new_loc["id"],
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        result = Environment().info({'name': name})
```
Comment (elyezer) [inline tests/foreman/cli/test_environment.py]: This info is not necessary, the factory already run info for you. You can drop lines 104-106 and refactor 107 to:

```python
self.assertIn(new_loc['name'], environment['locations'])
```

File: tests/foreman/cli/test_environment.py [hunk 3]
```diff
@@ -83,6 +83,52 @@ def test_list(self):
...
+        try:
+            new_org = make_org()
+            make_environment({
+                'name': name,
+                'organization-ids': new_org["id"],
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        result = Environment().info({'name': name})
```
Comment (elyezer) [inline tests/foreman/cli/test_environment.py]: Same as previous commits.

**Comment by elyezer** [comment]
ACK, pending comments.

**Comment by oshtaier** [comment]
Thanks for review. I amended the commit, but forgot to force push. Can I remove that unnecessary, empty commit?

**Comment by elyezer** [comment]
Looks good know, thanks.

APT

**Comment by omaciel** [comment]
ACK. Thank you!

**Pull request merged**